.github/workflows/linkchecker.yml

```diff
@@ -4,11 +4,19 @@ jobs:
   markdown-link-check:
     runs-on: ubuntu-latest
     steps:
+    - name: Wait for Vercel preview deployment to be ready
+      uses: patrickedqvist/wait-for-vercel-preview@master
+      id: waitForVercelPreviewDeployment
+      with:
+       token: ${{ secrets.GITHUB_TOKEN }}
+       max_timeout: 180 #seconds
+    - name: Set Deployment URL Env
+      run:
+        echo "DEPLOYMENT_URL=${{ steps.waitForVercelPreviewDeployment.outputs.url }}" >> $GITHUB_ENV
```

Comment:
Watch out if the deployment for this branch is not redeployed, then this fails:

maybe the https://github.com/UnlyEd/github-action-await-vercel works there as it waits for deployment to be ready ?

Reply:
Yes it should work, example : https://github.com/UnlyEd/next-right-now/blob/v4.0.12-v2-mst-aptd-at-lcz-sty/.github/workflows/deploy-vercel-staging.yml#L174-L236

Reply:
So close. I'll take a look at the other action. I wonder if we can set a long timeout and instead cancel the job after 3 minutes. Maybe even open a PR against the action to not fail for inactive deployments.

Reply:
I figured out a happy medium with the current setup. The job will now get canceled if there is no deployment. But since it is a cancel GitHub puts a red x by the SHA. But looking at the checks we can see that it is not a true failure.

maybe this could be a interim as I look further into the other action, which seems to require vercel tokens. Let me know what you think.